static/src/javascripts/projects/common/modules/ui/lazy-load-images.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import lozad from 'lozad';
+
+export const lazyLoadImages = (): void => {
+    const selector = '.lazy-load-img';
+
+    const observer = lozad(selector, {
+        rootMargin: '100px 0px', // load images before they are actually viewed
+        load: element => {
+            const isIE =
```

Comment:
I'm assuming this function block is run for each image? Could we move this `isIE` declaration out of this scope and up into the `lazyLoadImages` scope so we just run this line of code once? We only need to make this declaration once as it'll always be the same.